installer/targets/git-hooks/git_hooks/pre-commit.d/author

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+AUTHORINFO=$(git var GIT_AUTHOR_IDENT) || return 1
+NAME=$(printf '%s\n' "${AUTHORINFO}" | sed -n 's/^\(.*\) <.*$/\1/p')
+EMAIL=$(printf '%s\n' "${AUTHORINFO}" | sed -n 's/^.* <\(.*\)> .*$/\1/p')
+
+echo "I'm going to commit with \"$NAME <$EMAIL>\""
+if [[ "${NAME}" == "amigo" ]]
```

Comment:
A lot of athome laptop and also personal laptops use amigo as username. So this check will make me and a lot of people angry. So please convert to hostname or use the environment variable ROBOT_REAL.

Reply:
For the athome laptops this check still makes sense no?

But for the personal ones it does not...